src/configureWorkspace/configure.ts

```diff
@@ -113,19 +113,29 @@ function genDockerComposeDebug(serviceNameAndRelativePath: string, platform: Pla
 }
 
 function genDockerIgnoreFile(service: string, platformType: string, os: string, ports: number[]): string {
-    return `node_modules
-npm-debug.log
-Dockerfile*
-docker-compose*
-.dockerignore
-.git
-.gitignore
-.env
-*/bin
-*/obj
-README.md
-LICENSE
-.vscode`;
+    return `**/node_modules
+**/npm-debug.log
```

Comment:
Can we alphabetize this list? I use the [sort lines](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) extension in VS Code

Reply:
Certainly can!